src/storage/txn/flow_controller/singleton_flow_controller.rs

```diff
@@ -406,7 +406,7 @@ struct CfFlowChecker {
     // When the write flow is about 100MB/s, we observed that the compaction ops
     // is about 2.5, it means there are 750 compaction events in 5 minutes.
     long_term_pending_bytes:
-        Smoother<f64, 1024, SMOOTHER_STALE_RECORD_THRESHOLD, SMOOTHER_TIME_RANGE_THRESHOLD>,
+        Option<Smoother<f64, 1024, SMOOTHER_STALE_RECORD_THRESHOLD, SMOOTHER_TIME_RANGE_THRESHOLD>>,
```

Comment:
Why make it optional?

Reply:
Because multi-rocksdb's region level flow controller will not use it (it will only use L0 files as the factor of flow control per region)

And, we will have a global singleton_flow_controller for multi-rocksdb's flow control based on pending bytes.